__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+process.env.NODE_ENV = "test";
+
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body;
+        expect(topics).toBeInstanceOf(Array);
```

Comment:
What would happen here if there were some error that returned us with an empty array? It's still an array, so it would still hit the .forEach, and it wouldn't have anything to check whether or not it was a string, so it wouldn't return false - so we'd get a pass where we shouldn't. Please check for the length of the array before heading into your forEach.